### ISSUE TITLE
access to solver workflow 

### ISSUE THREAD

**Issue by gantnera** [ansys/pyfluent#2411]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

if running fluent solver you can access the workflow directly via the pyconsole, e.g.
workflow.InitializeWorkflow(WorkflowType=r"Turbo Workflow")

if running pyfluent externally I couldn´t figure out how to access this object, e.g.
solver.workflow.InitializeWorkflow(WorkflowType=r"Turbo Workflow")
leads to an error.

so how to access & control the workflow object?

### 📝 Steps to reproduce

launch fluent via pyfluent & execute command:
solver.workflow.InitializeWorkflow(WorkflowType=r"Turbo Workflow")

### 💻 Which operating system are you using?

Windows

### 📀 Which ANSYS version are you using?

v241

### 🐍 Which Python version are you using?

3.10

### 📦 Installed packages

```shell
Package                               Version
------------------------------------- ---------------
ansys-api-dbu                         0.2.0
ansys-api-dyna                        0.3.2
ansys-api-fluent                      0.3.18
ansys-api-geometry                    0.3.0
ansys-api-mapdl                       0.5.1
ansys-api-mechanical                  0.1.0
ansys-api-meshing-prime               0.1.1
ansys-api-platform-instancemanagement 1.0.0
ansys-api-pyensight                   0.3.2
ansys-api-sherlock                    0.1.22
ansys-api-systemcoupling              0.1.0
ansys-dpf-composites                  0.3.1
ansys-dpf-core                        0.9.0
ansys-dpf-gate                        0.4.1
ansys-dpf-gatebin                     0.3.1
ansys-dpf-post                        0.5.0
ansys-dyna-core                       0.4.2
ansys-dynamicreporting-core           0.5.0
ansys-fluent-core                     0.19.dev1
ansys-geometry-core                   0.4rc1
ansys-grantami-bomanalytics           1.2.0
ansys-grantami-bomanalytics-openapi   1.1.0
ansys-grantami-recordlists            1.0.0.post1
ansys-grantami-serverapi-openapi      1.0.0
ansys-grpc-dpf                        0.7.1
ansys-mapdl-core                      0.66.0
ansys-mapdl-reader                    0.52.17
ansys-math-core                       0.1.3
ansys-mechanical-core                 0.10.2
ansys-meshing-prime                   0.5.0.dev6
ansys-motorcad-core                   0.2.0
ansys-openapi-common                  1.2.2
ansys-optislang-core                  0.4.0
ansys-platform-instancemanagement     1.1.2
ansys-pyensight-core                  0.6.0
ansys-pythonnet                       3.1.0rc1
ansys-seascape                        0.2.0
ansys-sherlock-core                   0.4.0
ansys-systemcoupling-core             0.3.0
ansys-tools-path                      0.3.2
anyio                                 4.2.0
appdirs                               1.4.4
argon2-cffi                           23.1.0
argon2-cffi-bindings                  21.2.0
arrow                                 1.3.0
asgiref                               3.7.2
asttokens                             2.4.1
async-lru                             2.0.4
attrs                                 23.2.0
Babel                                 2.14.0
beartype                              0.16.4
beautifulsoup4                        4.12.3
bleach                                6.1.0
build                                 1.0.3
cachetools                            5.3.1
certifi                               2023.5.7
cffi                                  1.15.1
charset-normalizer                    3.1.0
click                                 8.1.3
clr-loader                            0.2.6
colorama                              0.4.6
comm                                  0.2.1
contourpy                             1.1.0
cryptography                          41.0.1
cycler                                0.11.0
debugpy                               1.8.0
decorator                             5.1.1
defusedxml                            0.7.1
Django                                4.2.7
docker                                6.1.3
docutils                              0.20.1
exceptiongroup                        1.2.0
executing                             2.0.1
fastjsonschema                        2.19.1
filelock                              3.13.1
fonttools                             4.40.0
fqdn                                  1.5.1
geomdl                                5.3.1
google-api-core                       2.11.1
google-api-python-client              2.92.0
google-auth                           2.21.0
google-auth-httplib2                  0.1.0
googleapis-common-protos              1.59.1
grpcio                                1.56.0
grpcio-health-checking                1.48.2
grpcio-status                         1.48.2
h5py                                  3.9.0
httplib2                              0.22.0
idna                                  3.4
importlib-metadata                    6.7.0
ipykernel                             6.29.0
ipython                               8.20.0
ipywidgets                            8.1.1
isoduration                           20.11.0
jaraco.classes                        3.3.0
jedi                                  0.19.1
Jinja2                                3.1.3
joblib                                1.3.1
json5                                 0.9.14
jsonpointer                           2.4
jsonschema                            4.21.1
jsonschema-specifications             2023.12.1
jupyter                               1.0.0
jupyter_client                        8.6.0
jupyter-console                       6.6.3
jupyter_core                          5.7.1
jupyter-events                        0.9.0
jupyter-lsp                           2.2.2
jupyter_server                        2.12.5
jupyter_server_terminals              0.5.2
jupyterlab                            4.0.12
jupyterlab_pygments                   0.3.0
jupyterlab_server                     2.25.2
jupyterlab-widgets                    3.0.9
keyring                               24.3.0
kiwisolver                            1.4.4
lxml                                  4.9.3
markdown-it-py                        3.0.0
MarkupSafe                            2.1.4
matplotlib                            3.7.2
matplotlib-inline                     0.1.6
mdurl                                 0.1.2
mistune                               3.0.2
more-itertools                        10.1.0
nbclient                              0.9.0
nbconvert                             7.14.2
nbformat                              5.9.2
nest-asyncio                          1.6.0
nh3                                   0.2.14
notebook                              7.0.7
notebook_shim                         0.2.3
numpy                                 1.25.0
overrides                             7.7.0
packaging                             23.1
pandas                                2.0.3
pandocfilters                         1.5.1
parso                                 0.8.3
Pillow                                10.0.0
Pint                                  0.22
pip                                   23.3.2
pkginfo                               1.9.6
platformdirs                          3.8.0
plumbum                               1.8.2
pooch                                 1.7.0
prometheus-client                     0.19.0
prompt-toolkit                        3.0.43
protobuf                              3.20.3
protoc-gen-swagger                    0.1.0
psutil                                5.9.5
pure-eval                             0.2.2
pyaedt                                0.6.94
pyansys                               2024.1rc1
pyansys-docker                        5.0.4
pyansys-tools-versioning              0.4.0
pyasn1                                0.5.0
pyasn1-modules                        0.3.0
pycparser                             2.21
Pygments                              2.16.1
pyiges                                0.3.1
pyparsing                             3.0.9
pypiwin32                             223
pypng                                 0.20220715.0
pyproject_hooks                       1.0.0
pyspnego                              0.9.1
python-dateutil                       2.8.2
python-json-logger                    2.0.7
pythonnet                             3.0.2
pytwin                                0.5.0
pytz                                  2023.3
pyvista                               0.43.2
pywin32                               306
pywin32-ctypes                        0.2.2
pywinpty                              2.0.12
PyYAML                                6.0
pyzmq                                 25.1.2
qtconsole                             5.5.1
QtPy                                  2.4.1
readme-renderer                       42.0
referencing                           0.33.0
requests                              2.31.0
requests-negotiate-sspi               0.5.2
requests-ntlm                         1.2.0
requests-toolbelt                     1.0.0
rfc3339-validator                     0.1.4
rfc3986                               2.0.0
rfc3986-validator                     0.1.1
rich                                  13.7.0
rpds-py                               0.17.1
rpyc                                  5.3.1
rsa                                   4.9
scikit-learn                          1.3.0
scipy                                 1.11.1
scooby                                0.7.2
Send2Trash                            1.8.2
setuptools                            65.5.0
six                                   1.16.0
sniffio                               1.3.0
soupsieve                             2.5
sqlparse                              0.4.4
stack-data                            0.6.3
terminado                             0.18.0
threadpoolctl                         3.1.0
tinycss2                              1.2.1
tomli                                 2.0.1
tornado                               6.4
tqdm                                  4.65.0
traitlets                             5.14.1
twine                                 4.0.2
types-python-dateutil                 2.8.19.20240106
typing                                3.7.4.3
typing_extensions                     4.8.0
tzdata                                2023.3
uri-template                          1.3.0
uritemplate                           4.1.1
urllib3                               1.26.16
vtk                                   9.2.6
wcwidth                               0.2.13
webcolors                             1.13
webencodings                          0.5.1
websocket-client                      1.6.3
widgetsnbextension                    4.0.9
zipp                                  3.15.0
```

**Comment by seanpearsonuk** [comment]
![image](https://github.com/ansys/pyfluent/assets/93727996/92f4ca6a-575a-4dae-9b81-6c3d7a06019a)

This works. @raph-luc, Is it working for you?

**Comment by raph-luc** [comment]
You're right @seanpearsonuk, it works for me as well. Tried both v241 and v242 and both work fine on my end.
![image](https://github.com/ansys/pyfluent/assets/132297401/de87ca8f-0ff2-4daa-aca0-06245fd5fc3f)

@gantnera I am not sure what may be going wrong. Can you try updating PyFluent to the latest 0.19.2 version? You are on a development version `0.19.dev1` according to your installed packages logs.

**Comment by raph-luc** [comment]
User confirmed the issue is solved with 0.19.2
